### PR TITLE
move lambda tools to user guides

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

File: content/en/user-guide/lambda-tools/_index.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-weight: 10
+weight: 45
 description: >
   Develop your Lambdas more efficiently.
 aliases:
   - /tools/lambda-tools/
+  - /user-guide/tools/lambda-tools/
 ---
 
 Lambdas are awesome!
```
Comment (joe4dev) [inline content/en/user-guide/lambda-tools/_index.md:12]: That should be more informative ;)

File: content/en/user-guide/lambda-tools/lambda-vscode-extension/index.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
 ---
 title: "Lambda VSCode Extension"
```
Comment (joe4dev) [inline content/en/user-guide/lambda-tools/lambda-vscode-extension/index.md:2]: The "Lambda" prefix is not really needed here

Comment (joe4dev) [reply content/en/user-guide/lambda-tools/lambda-vscode-extension/index.md:2]: same in the text below

Comment (joe4dev) [inline content/en/user-guide/lambda-tools/lambda-vscode-extension/index.md:2]: IMHO, the "LocalStack VSCode Extension" would fit better under "LocalStack Tools". In its current state, it is indeed very limited and only relevant for Lambda, I guess that's why it ended up under the "Lambda Tools":

Screenshot 2024-01-02 at 19 14 06

**Review by joe4dev — APPROVED** [review]
I added a few comments/suggestions but it looks fine overall 👍 
It helps that the "Lambda Tools" are right below the "LocalStack Tools" such that looking for them under "LocalStack Tools" likely leads to finding them anyway. Even if alphabetically ordered.

File: content/en/user-guide/lambda-tools/lambda-vscode-extension/index.md (hunk 2 above)
Comment (HarshCasper) [reply content/en/user-guide/lambda-tools/lambda-vscode-extension/index.md:2]: I believe we can still keep it around in `Lambda Tools` given that it only focusses on improving the Lambda Development experience right now.

**Review by dfangl — APPROVED** [review]
I do not have a definite opinion about the moving of the pages one level up, but it is fine by me!

**Pull request merged**